build/build/src/enso.rs

```diff
@@ -29,7 +29,7 @@ impl From<bool> for Boolean {
 }
 
 ide_ci::define_env_var! {
-    ENSO_JVM_OPTS, String;
```

Comment:
`ENSO_JVM_OPTS` works only for project-manager or for launcher, but not when we run Enso via built engine distribution. For that, we need to pass the JVM flags via `JAVA_OPTS`.